site/gatsby-site/package.json

```diff
@@ -23,6 +23,7 @@
     "@postlight/mercury-parser": "^2.2.1",
     "algoliasearch": "^4.10.5",
     "async": "^3.2.2",
+    "babel-plugin-lodash": "^3.3.4",
```

Comment:
I don't see any reference to this package in the code. If this is not used please remove and re-generate the `package-lock.json` file.

Reply:
removed babel-plugin-lodash